free-2.md

```diff
@@ -11,41 +11,62 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+Accordion music is generally written on a regular grand staff, with keyboard notes written in the treble clef and bass notes/Stradella chords written in the bass clef. However, a great deal is left out in these directions, like when to reverse bellows direction, or sort of tacked on top, like the notation for which chord to play, when to switch registers, bellows shake, and noise-sounds.
```

Comment:
Really cool domain! I think that this project would be the biggest language challenge of your three proposals - there's a lot that goes into music, and describing all that could get expansive fast. So you'd definitely have your hands full there!